core/src/br/cefetmg/games/screens/BaseScreen.java

```diff
@@ -186,22 +190,21 @@ public void resize(int width, int height) {
      * chamada.
      */
     @Override
-    public final void render(float dt) {
-        if (assets.update()) {
-            if (messagesFont == null) {
+    public final void render(float dt)
+    {
+        if (assets.update() && isLoadingOver) {
+            if (messagesFont == null)
                 messagesFont = assets.get("snacker-comic-50.ttf");
-            }
             // chama função para gerenciar o input
             handleInput();
-            
+
             // chama função para atualizar a lógica da tela
             update(dt);
 
             // a tela pode ter sido "disposed" durante este último update, então
             // verificamos se isso aconteceu para saber se seguimos adiante
-            if (wasJustDisposed) {
+            if (wasJustDisposed)
```

Comment:
@Candrade175 e @gustavojordao, em termos de Engenharia de Software, é importante sempre usar chaves para criar um bloco mesmo quando temos apenas uma instrução dentro de um if/else/for/while. Vejam esta [resposta no StackOverflow com os motivos](https://stackoverflow.com/a/12193202/1783793).